h/celery.py

```diff
@@ -38,7 +38,6 @@
     task_routes={
         "h.tasks.indexer.add_annotation": "indexer",
         "h.tasks.indexer.delete_annotation": "indexer",
-        "h.tasks.indexer.reindex_user_annotations": "indexer",
```

Comment:
Do we know what this is supposed to do? It makes me a bit nervous to remove all our indexing tasks from it without understanding why.

Reply:
It routes things to the `"indexer"` queue which, unlike the default queue, has `durable=True` (meaning that Celery/Rabbit make the tasks/messages durable against Rabbit restarts by persisting them to disk). `reindex_user_annotations` has been deleted

Reply:
I've added `add_annotations_between_times` and `add_users_annotations`. Especially when renaming a user I think we do want this Celery task to be durable, otherwise they could get renamed in the DB but not in the search index.

For some reason it looks like the task for sending emails is not added to a durable queue either, which seems like a bad idea. I'm not 100% sure though

Reply:
I'm not 100% sure whether this makes any difference or not